### ISSUE TITLE
How to apply custom mask for cel-phone type?

### ISSUE THREAD

**Issue by nurbek-ab** [benhurott/react-native-masked-text#26]
I want to do something like this:
```
<TextInputMask
            type={'cel-phone'}
            placeholder="Enter your mobile number"
            placeholderTextColor="#cccbd0"
            style={styles.numberInput}
            options={{
              mask: '(999) 999-9999'
            }}
/>
```
But it doesn't work that way, so I did this instead:
```
<TextInputMask
            type={'custom'}
            placeholder="Enter your mobile number"
            placeholderTextColor="#cccbd0"
            style={styles.numberInput}
            options={{
              mask: '(999) 999-9999'
            }}
/>
```
Seems to be ok but it opens alphabetical keyboard. Is it possible to use type="cel-phone" with a custom mask?

**Comment by benhurott** [comment]
Hi!

I fix it at 1.5.2.

You now can use `keyboardType="numeric"` for example.

Thanks for reporting =).

**Comment by nurbek-ab** [comment]
Thank you! Great job.
